source/tests/system/nidigital_driver_api_tests.cpp

```diff
@@ -0,0 +1,332 @@
+#include <gtest/gtest.h>
+
+#include "nidigital/nidigital_library.h"
+#include "nidigital/nidigital_service.h"
+
+namespace ni {
+namespace tests {
+namespace system {
+
+namespace digital = nidigital_grpc;
+
+const int kDigitalDriverApiSuccess = 0;
+
+class NiDigitalDriverApiTest : public ::testing::Test {
+ protected:
+  NiDigitalDriverApiTest()
+  {
+    ::grpc::ServerBuilder builder;
```

Comment:
[FYI] This doesn't need immediate action but when you merge into main or if you update the feature branch to pull in recent commits from main you'll likely want to update these system tests. 

There's a PR in the main repo right now that's cleaning up a lot of this boiler plate setup code so eventually we can update these new tests to follow the pattern other system tests adopted in this PR: https://github.com/ni/grpc-device/pull/196.

Reply:
Thanks @reckenro for bringing this to notice. I have added task for tracking that.